homeassistant/components/shelly/config_flow.py

```diff
@@ -107,15 +119,18 @@ async def validate_input(
         CONF_SLEEP_PERIOD: get_block_device_sleep_period(block_device.settings),
         "model": block_device.model,
         CONF_GEN: gen,
+        CONF_PORT: port,
```

Comment:
Why are you returning the input `port` as an output? the caller already has the `port`? (we are also not returning the `host` here and they are used together
```suggestion
```

Reply:
Probably a leftover when port was parsed.